boards/holybro/durandal-v1/nuttx-config/nsh/defconfig

```diff
@@ -215,35 +219,29 @@ CONFIG_TIME_EXTENDED=y
 CONFIG_TTY_SIGINT=y
 CONFIG_TTY_SIGSTP=y
 CONFIG_UART4_BAUD=57600
-CONFIG_UART4_DMA=y
```

Comment:
These were removed automatically by menuconfig (qconfig).

Reply:
This would be RX and TX DMA now.

Reply:
STM32H7 doesn't have serial DMA.